org-formation/organization.yaml

```diff
@@ -18,16 +18,23 @@ Organization:
     Properties:
       DefaultOrganizationAccessRoleName: OrganizationAccountAccessRole
 
-  ScienceOU:
+  ScienceDevOU:
     Type: OC::ORG::OrganizationalUnit
     Properties:
-      OrganizationalUnitName: science
+      OrganizationalUnitName: ScienceDev
       Accounts:
         - !Ref ScipoolDevAccount
-        - !Ref ScipoolProdAccount
-        - !Ref ScicompAccount
         - !Ref SandboxAccount
         - !Ref MobileHealthDataEngineeringAccount
+        - !Ref SageITAccount
```

Comment:
Should SageIT really be in this OU?

Reply:
SageITAccount currently has resources in us-west region and so that's why it's in this group.   Please propose something if you would like to see something different.